Riot/Modules/Common/Recents/Views/RootTabEmptyView.swift

```diff
@@ -50,14 +60,25 @@ final class RootTabEmptyView: UIView, NibLoadable {
         super.awakeFromNib()
         
         self.informationLabel.text = VectorL10n.homeEmptyViewInformation
+        
+        self.iconBackgroundView.layer.masksToBounds = true
+        self.iconBackgroundView.layer.cornerRadius = self.iconBackgroundView.bounds.width / 2
```

Comment:
Do we need to update this on `layoutSubviews`?

Reply:
Well, in this case we don't as the size of the view is fixed.